Config_Models/example_model/model.hpp

```diff
@@ -2,9 +2,9 @@
 #define ZPSW3_CONFIG_HPP
 
```

Comment:
'#include fbwa.hpp' may need to be added here. Without it, I get a linting error on line 121.